c++/src/capnp/fuzz-run.c++

```diff
@@ -0,0 +1,21 @@
+#include "test-util.h"
+#include <kj/main.h>
+#include "serialize.h"
+#include <capnp/test.capnp.h>
+#include <unistd.h>
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t* Data, size_t Size)
+{
```

Comment:
Can we follow KJ/capnp formatting conventions here, like putting opening braces at the end of the preceding line, and using two-space indents?

Reply:
ok, let me know if there is more to do